src/index.js

```diff
@@ -42,6 +42,8 @@ function render() {
     quoteFilename(outputFilename),
   ].join(" ");
 
+  document.getElementById("Copy").textContent="Copy"; //reset copy button text on input change.
```

Comment:
Instead of using selectors, I just gave the elements global IDs—you can see what I did elsewhere in this file. Could you do that instead?

Reply:
Hi Evan,

Could you explain how global IDs works.
Btw, I have implemented it.